translator/internaldata/resource_to_oc.go

```diff
@@ -79,7 +79,7 @@ func getSDKLangToOCLangCodeMap() map[string]int32 {
 }
 
 func internalResourceToOC(resource pdata.Resource) (*occommon.Node, *ocresource.Resource) {
-	if resource.IsNil() {
+	if resource.IsEmpty() {
 		return nil, nil
 	}
```

Comment:
Can be deleted or substituted by `if attrs.Len() == 0` check that we have below if want to return nil in that case.

Reply:
Done.